doc/user_manual/runInfo.tex

```diff
@@ -59,6 +59,11 @@ \subsection{RunInfo: Input of Calculation Flow}
   %
   \default{mpiexec}
 
+  %%%%%% N THREADS
+\item \xmlNode{threadCommand}, \xmlDesc{string, optional field}, specifies the command used to set the number of threads.
+  %
+  \default{--n-threads=}
```

Comment:
why do you need to keep "="?

Reply:
the problem is that not all the "thread commands" need an "="... So if I want to use, for example, ```-omp 32``` I can not if we keep the "equal".

Reply:
The problem is, in the code, you do keep another equal sign (see below): 
![image](https://user-images.githubusercontent.com/7321071/122972047-f36e4f00-d34c-11eb-90ce-dd23108379d2.png)

Reply:
Oh good catch. But it is not a problem...it is simply a typo :)

Reply:
Why not just make the default ```--n-thread=%NUM_CPUS%```  and then the user can put %NUM_CPUS% where ever they want and put spaces or = or whatever.